dbt/include/athena/macros/adapters/relation.sql

```diff
@@ -20,3 +20,9 @@
     alter table {{ relation.render_hive() }} set tblproperties ('classification' = '{{ format }}')
   {%- endcall %}
 {%- endmacro %}
+
+{% macro athena__rename_relation(from_relation, to_relation) %}
+  {% call statement('rename_relation') -%}
+    alter table {{ from_relation.render_hive() }} rename to `{{ to_relation.identifier }}`
```

Comment:
why do we have inconsistency between how the from_relation and to_relation is quoted? isn't possible to use `to_relation.identifier.render_hive() ` as well without backticks?

Reply:
Because we cannot have the name of the schema in the rename to part, it should only be the name of the table.

The `render_hive` function seems to apply to the `Relation` object therefore this is not possible.

Reply:
https://docs.aws.amazon.com/athena/latest/ug/querying-iceberg-managing-tables.html#querying-iceberg-alter-table-rename
> ALTER TABLE my_db.my_table RENAME TO my_db2.my_table2

Seems that we can put schema name based on what the docs say.

Reply:
@Jrmyy did you have a look at this comment?

Reply:
I added the database but because of the new quoting policy including the data catalog, we cannot use the `render_hive` because otherwise I have this error:

>  backquoted identifiers are not supported; use double quotes to quote identifiers